numpyro/infer/elbo.py

```diff
@@ -33,6 +34,8 @@ class ELBO:
 
     :param num_particles: The number of particles/samples used to form the ELBO
         (gradient) estimators.
+    :param vectorize_particles: Whether to use `jax.vmap` to obtain elbos over the
```

Comment:
is this only for eval or also for training?

Reply:
this is for both.

Reply:
nit:
> :param vectorize_particles: Whether to use `jax.vmap` to compute ELBOs over the `num_particles`-many particles in parallel. If False use `jax.lax.map`. Defaults to True.